### PR TITLE
fix(bower.json): change main field to target the dist file

### PR DESCRIPTION
**Pull request by ghost**
CMIIW

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "localforage-cordovasqlitedriver",
   "version": "1.4.1",
   "main": [
-    "src/localforage-cordovasqlitedriver.js"
+    "dist/localforage-cordovasqlitedriver.js"
```
Comment (thgreasi) [inline bower.json:5]: You are totally right. Nice catch :+1: I probably forgot to change it while migrating to rollup.

File: bower.json [hunk 2]
```diff
@@ -18,6 +18,6 @@
     "site*"
   ],
   "dependencies": {
-    "localforage": "^1.4.0"
+    "localforage": "~1.4.0"
```
Comment (thgreasi) [inline bower.json]: I would prefer not to change this. I'm confident that the API of localforage v1 will not dramatically change. In other words, I think we will face no compatibility issues with localforage v1.5 .

**Comment by thgreasi** [comment]
Please just revert the dependency version change and we can merge this "as is" (prefer not to do it myself so you can have all the credits).

**Comment by nueko** [comment]
Ah, I don't  care about the credit.
but i will try.

**Pull request merged**